uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -695,9 +790,13 @@ sub dicom_to_minc {
         close MAIL;
         croak("dicom_to_minc failure, exit code $exit_code");
```

Comment:
Yep, this should be spooled IMHO. This way when the notification tables are used to forward the pipeline output back to the user's web browser, this message will appear on the webpage.